app/controllers/customer_resources_controller.rb

```diff
@@ -0,0 +1,32 @@
+class CustomerResourcesController < ApplicationController
+
+  def show
+    vendor_id, package_id, title_id = params[:id].split('-')
+
+    customer_resource_path = "vendors/%{vendor_id}/packages/%{package_id}/titles/%{title_id}" % {
+      vendor_id: vendor_id || 0,
+      package_id: package_id || 0,
+      title_id: title_id || 0
+    }
+    # Make the request for the title from the RM API
+    response = rmapi.request(:get, customer_resource_path)
```

Comment:
Can we push this down to the model layer?

```ruby
resource = CustomerResource.find(params[:id])
render jsonapi: resource, include params[:include]
```

We may have to add `rmapi` service to the request context for that. In general, serialization / deserialization from the _underlying_ network layer (usually postgres server, but in our case rmapi server) should be handled by the model layer, no the HTTP layer.

Reply:
indeed we can, and we should probably apply this pattern to other resource types as well (though in another PR)